components/MapChart/MapChartHeader/MapHeaderContainer.jsx

```diff
@@ -96,7 +101,7 @@ const MapHeaderContainer = ({
   countryData = [],
   ...props
 }) => {
-  const [selectedYear, setSelectedYear] = useState('All')
+  const [selectedYear, setSelectedYear] = useState('2021')
```

Comment:
could we set the default to the most recent year data available?

Reply:
I need to check if there is a good way to take care of this, it's a bit of a crappy situation with the default on the first render where potentially the data is not in yet.